tests/estimators/test_finite_difference.py

```
@@ -0,0 +1,186 @@
+
+        config = FiniteDifferenceConfig(step_size=1e-3, num_samples=3000)
+        fd_grad = finite_difference_gradient(linear_problem, theta, config)
+        assert jnp.allclose(fd_grad, 1.0, atol=0.05)
+
+    def test_binary_choice_gradient_half(self, binary_problem):
+        """FD should produce gradient near 0.5 at θ=0.
+
+        Analytical solution:
+        - π₁ = exp(2θ)/(exp(2θ)+1)
+        - dπ₁/dθ at θ=0 = 2/(2²) = 0.5
+        - dE/dθ = dπ₁/dθ × (f₁-f₂) = 0.5 × 1 = 0.5
```

Ruff (0.14.1)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)